libs/ledgerjs/packages/devices/src/index.ts

```diff
@@ -83,7 +83,7 @@ const devices: { [key in DeviceModelId]: DeviceModel } = {
   },
   [DeviceModelId.nanoFTS]: {
     id: DeviceModelId.nanoFTS,
```

Comment:
should we rename now the `nanoFTS` into `stax` in `DeviceModelId.nanoFTS` ? Or wait another PR maybe ?

Reply:
I'd say another PR

Reply:
Also I don't know if we should actually do it at all as some 3 party implementations might already use it, so it'd be a breaking change (which is ok if versioning is done properly but not my call)